ui/src/components/containers/ContainerInspect.js

```diff
@@ -11,7 +11,10 @@ import DateTime from 'react-datetime';
 import '../../../node_modules/react-datetime/css/react-datetime.css';
 
 import { logsContainer, execContainer, topContainer } from '../../api';
-import { shortenImageName } from '../../lib';
+import { shortenImageName, showError, showSuccess } from '../../lib';
```

Comment:
ShortenImageName is used on line 589! My Phpstorm gives notice of unused imports so no need for the lint?!